tests/db_modes_test.py

```diff
@@ -42,10 +42,11 @@ def run_and_kill(extraCmd="", killsig=signal.SIGTERM, preCmd=""):
             tries = tries - 1
             port = port + 1
     # now start nodeos
+    time.sleep(5)
```

Comment:
Is this intended to be part of the PR?

Reply:
Yes. This PR morphed into get the tests passing PR.